src/app/store/effects/VehicleTestResultModel.effects.ts

```diff
@@ -9,7 +9,8 @@ import {
   UpdateSelectedTestResultModelSuccess,
   UpdateTestResult,
```

Comment:
Please update this effect to send testResultId instead of systemNumber when calling the service. We need to be consistent with 14406 BE ticket